src/components/PopoverMenu.tsx

```diff
@@ -317,7 +324,7 @@ PopoverMenu.displayName = 'PopoverMenu';
 export default React.memo(
     PopoverMenu,
     (prevProps, nextProps) =>
-        prevProps.menuItems.length === nextProps.menuItems.length &&
+        prevProps.menuItems === nextProps.menuItems &&
```

Comment:
Is this still needed?

Reply:
Yes, it needs to show the error when the connection is failure.